### ISSUE TITLE
Partially working on Acer A0751h

### ISSUE THREAD

**Issue by hjnijlunsing** [Baron-von-Riedesel/VSBHDA#1]
I couldn't get the original SBEMUb3 to work on my Acer A0751h; however this forks runs and I am able to run Epic Pinball with sound!

However two interesting observations:
- It _only_ works if I ENABLE my Network Boot in my BIOS; (to me this is really weird...); my only guess is that enabling this option ROM also initializes the Sound Card. MPXPlay works fine without this change.
- All tables work with sound EXCEPT Android (again to me this is also weird as I would expect all the tables would behave identically)

Thanks for your work!

**Comment by Baron-von-Riedesel** [comment]
> I couldn't get the original SBEMUb3 to work on my Acer A0751h; however this forks runs and ...

Is the sound HW of type "Intel HDA"?

> It only works if I ENABLE my Network Boot in my BIOS; [...] MPXPlay works fine without this change.

Really bizarre ... if MPXPlay works, VSBHDA (and SBEMU) should in any case work as well. I have no idea how to fix this.

Perhaps the network boot ROM changes the IRQ of the sound device? Perhaps comparing the status of the PCI devices with/without that ROM might give a glue...

**Comment by hjnijlunsing** [comment]
Yes the sound HW is of type "Intel HDA", with network boot disabled VSBHDA runs and detects sound card, however I do not get any sound when starting Epic Pinball.

If you are able to give me the commands to compare I would be happy to try it out!

**Comment by Baron-von-Riedesel** [comment]
> If you are able to give me the commands to compare I would be happy to try it out!

There exist a lot of tools that scan the PCI devices ( one is attached ). I don't know if it will help to find the problem, though.

[pciinfo.zip](https://github.com/Baron-von-Riedesel/VSBHDA/files/11981779/pciinfo.zip)

**Comment by hjnijlunsing** [comment]
Runnning PCIINFO hangs the system ;-)

However before it hangs I get for the HDA Device with Network Boot Enabled I get:
0 1b 0 4 3 0 7 8086/811b 10 multimedia controller (HDA)

Without network boot:
0 1b 0 4 3 0 7 8086/811b <EMPTY> multimedia controller (HDA)

Thus indeed you are right; enabling network boot gives IRQ 10 to the HDA device... Weird...
